src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SendApplicationIssueNotifications.java

```diff
@@ -37,7 +37,9 @@
                 }
             } else {
                 applicationIssuedNotification.sendToJointApplicant1(caseData, caseId);
-                applicationIssuedNotification.sendToJointApplicant2(caseData, caseId);
+                if (caseData.getApplication().getApplicant1KnowsApplicant2EmailAddress().toBoolean()) {
+                    applicationIssuedNotification.sendToJointApplicant2(caseData, caseId);
```

Comment:
Could getApplicant1KnowsApplicant2EmailAddress() ever return null?

Reply:
To get to this point they would have had to ask this question, but it never hurts to be defensive with our programming so will add a check 👍